src/Symfony/Component/Notifier/Bridge/Slack/Tests/SlackOptionsTest.php

```diff
@@ -187,4 +189,25 @@ public function fromNotificationProvider(): iterable
             (new Notification($subject))->emoji($emoji)->content($content),
         ];
     }
+
+    public function testConstructThrowsWithTooManyBlocks()
+    {
+        $this->expectException(LogicException::class);
+        $this->expectExceptionMessage('Maximum number of "blocks" has been reached (50).');
+
+        new SlackOptions(['blocks' => array_map(static function () { return ['type' => 'divider']; }, range(0, 50))]);
```

Comment:
this assert that 51 blocks throws an exception.

Could you please add a test that asserts that 50 are ok?